ui-react/src/eventsBridge.tsx

```diff
@@ -39,3 +39,19 @@ export async function toggleUIVersion() {
 export async function openExternal(url: string) {
   await electron.shell.openExternal(url);
 }
+
+export async function getLastLines(numberOfLines: number) {
```

Comment:
I suggest grouping the log related information into one function that returns an object with the log related info rather than having a separate function for each piece of data